anitya/db/migrations/versions/9262ccab3573_add_check_times.py

```diff
@@ -0,0 +1,42 @@
+"""Add check times
+
+Revision ID: 9262ccab3573
+Revises: 7a8c4aa92678
+Create Date: 2018-10-10 10:11:19.281233
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import arrow
+
+
+# revision identifiers, used by Alembic.
+revision = '9262ccab3573'
+down_revision = 'b13662e5d288'
+
+
+def upgrade():
+    """ Add next_check and last_check columns to the projects table. """
+    op.add_column(
+        'projects',
+        sa.Column(
```

Comment:
I don't see indexes being added or removed here -  did you regenerate your migration after altering the model to include an index?

Reply:
I forgot about that